modules/server/src/main/ts/BedrockFramework.ts

```diff
@@ -23,17 +23,13 @@ export const go = (settings: BedrockFrameworkSettings): void => {
   }).then((driver) => {
     const webdriver = driver.webdriver;
     const serveSettings: Serve.ServeSettings = {
-      projectdir: settings.projectdir,
-      basedir: settings.basedir,
+      ...settings,
       driver: Attempt.passed(webdriver),
       testfiles: [],
       master,
       runner,
-      loglevel: settings.loglevel,
-      customRoutes: settings.customRoutes,
+      customRoutes: undefined,
```

Comment:
This doesn't look right.

Reply:
This is correct, it's that the types were incorrect as `settings.customRoutes` was always `undefined` as it's not a valid setting for framework mode. See: https://github.com/tinymce/bedrock/blob/master/modules/server/src/main/ts/bedrock/cli/Clis.ts#L59

Reply:
As discussed on slack I've added a comment to state why we are passing undefined.